utils/doclint/generateApiJson.js

```diff
@@ -92,18 +92,19 @@ function sanitize(result) {
 
 /**
  * @param {Documentation.Type} type
+ * @param {boolean} sortProperties
  */
-function serializeType(type) {
+function serializeType(type, sortProperties = false) {
   /** @type {any} */
   const result = { ...type };
   if (type.properties)
-    result.properties = type.properties.map(serializeProperty);
+    result.properties = (sortProperties ? type.sortedProperties() : type.properties).map(serializeProperty);
```

Comment:
You also need to update [type generator](https://github.com/microsoft/playwright/blob/master/utils/generate_types/index.js), see the linter failures. Also let's roll these files to https://github.com/microsoft/playwright.dev/tree/master/src and make sure it produces correct results (we may need to update generator there as well)

Reply:
I was just taking a look at it. Thanks for the hint.

Reply:
How about now @yury-s?

I ran `SRC_DIR=~/code/playwright npm run roll` on playwright.dev but I got no changes. Maybe you can give me a hand when this one is merged.

Reply:
Absolutely